src/Controllers/NotificationType.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
```

Comment:
add `declare(strict_types=1);` in new files

Reply:
https://github.com/engelsystem/engelsystem/pull/1055 is not merged so :shrug: